index.html

```diff
@@ -1,13 +1,307 @@
+          </ul>
+        </div>
+      </div>
+      <div class="header-bottom row-style">
+        <a href="#">
+          <img class="logo-image" src="images/Suli forum (1).png" alt="logo" />
+        </a>
+        <nav class="navbar navbar-expand-lg bg-white">
+          <div class="container px-0">
+            <button
+              class="navbar-toggler"
+              type="button"
+              data-bs-toggle="collapse"
+              data-bs-target="#navbarNav"
+              aria-controls="navbarNav"
+              aria-expanded="false"
+              aria-label="Toggle navigation"
+            >
+              <span class="navbar-toggler-icon"></span>
+            </button>
+            <div class="collapse navbar-collapse" id="navbarNav">
+              <ul class="navbar-nav ms-auto gap-5">
+                <li class="nav-item">
+                  <a class="nav-link" href="about.html">About</a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link" href="#program">Program</a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link" href="#sponsors">Sponsors</a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link" href="#news">News</a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link" href="#attend">Attend</a>
+                </li>
+              </ul>
+            </div>
+          </div>
+        </nav>
+      </div>
+    </header>
+
+    <main class="main">
+      <section class="section-hero">
+        <div class="hero-container row-style">
+          <h4 class="hero-title-text">Toward a New Regional Order</h4>
+          <h1 class="hero-heading">THE SEVENTH ANNUAL SULAIMANI FORUM</h1>
+          <p class="hero-text margin-medium">
+            The Sulaimani Forum is the signature annual event hosted by the
+            Institute of Regional and International Studies (IRIS) at AUIS.
+          </p>
+          <div class="event-details hero-title-text">
+            <p>8 <sup>th</sup> - 10 <sup>th</sup> DEC 2022</p>
+            <a href="#">
+              <svg
+                xmlns="http://www.w3.org/2000/svg"
+                width="16"
+                height="16"
+                fill="currentColor"
+                class="bi bi-geo-alt-fill"
+                viewBox="0 0 16 16"
+              >
+                <path
+                  d="M8 16s6-5.686 6-10A6 6 0 0 0 2 6c0 4.314 6 10 6 10zm0-7a3 3 0 1 1 0-6 3 3 0 0 1 0 6z"
+                />
+              </svg>
+              Kurdistan, Sulaimani
+            </a>
+          </div>
+          <div class="header-buttons">
+            <button type="button" class="hero-button attend-button">
+              Attend
+            </button>
+            <button type="button" class="hero-button sponsor-button">
+              Sponsor
+            </button>
+          </div>
+        </div>
```

Comment:
While you did an amazing job working on the project requirements. it would be nice if you stick to design requirements. Please use the correct font size and font colors for the content in the home page.

This is your design:

![home page ](https://user-images.githubusercontent.com/79506105/198345263-e41893e1-d4a5-4eee-ab31-23ccbc40a6c2.png)

Please your design should look something similar to this:

![Original design](https://user-images.githubusercontent.com/79506105/198345463-7c9fbdee-2def-483c-a224-516eb65f3397.png)


2. Secondly I noticed whenn I switch to the mobile version of your project your 2 navigation bars are visible even your logo is visibile this is not supposed to be seen in the mobile version I also noticed your hamburger button is on the right hand side of your page it would be nice ifyou place it on the left hand side of your web page in mobile view.
This is your design: 

![mobile view ](https://user-images.githubusercontent.com/79506105/198346460-69a6ef77-ff02-4652-9707-865568b438e8.png)

Please follow this design:
![Screen Shot 2022-10-27 at 18 28 52](https://user-images.githubusercontent.com/79506105/198347063-20e4a8f7-1090-49f8-a70e-225ba2197394.png)